app/views/my_facilities/blood_pressure_control.html.erb

```diff
@@ -95,7 +95,7 @@
           <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="missed" colspan="2">No BP taken<br>in <%= visited_in_period %>
           </th>
           <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-percent" colspan="2">Controlled BP<br>in last 90 days</th>
-          <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-registered">Registered<br>&gt; 2 months ago</th>
+          <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-registered">Registered<br>&gt; 3 months ago</th>
```

Comment:
https://github.com/simpledotorg/simple-server/pull/1206/files#diff-a60da1211f63a657a368282d310b62f5R157


This footnote down below also needs change.

Reply:
Addressed in 5b0999e.